hadoop-ozone/dist/src/main/smoketest/admincli/scmrole.robot

```diff
@@ -30,4 +30,8 @@ Run scm roles
 List scm roles as JSON
     ${output} =         Execute          ozone admin scm roles --json
     ${leader} =         Execute          echo '${output}' | jq -r '.[] | select(.raftPeerRole == "LEADER")'
-                        Should Not Be Equal       ${leader}       ${EMPTY}
\ No newline at end of file
+                        Should Not Be Equal       ${leader}       ${EMPTY}
+
+List scm roles as TABLE
+    ${output} =         Execute          ozone admin scm roles --table
+                        Should contain   ${output}   LEADER
```

Comment:
Maybe we could add `|` to show that it is a table, like `Should contain ${output} | LEADER |`

Reply:
@whbing If it's JSON, should we confirm the presence of characters like`{`, `}` and `:`among others?